lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -112,7 +112,7 @@ sub test_network_interface {
     #flag SRIOV test as it need not restart network service
     my $is_sriov_test = "false";
     $is_sriov_test = "true" if caller 0 eq 'sriov_network_card_pci_passthrough';
-    script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180) if (is_sle('=11-sp4') || ($routed == 1) || ($isolated == 1));
+    script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180);
```

Comment:
Skipping this won't help you at all as the very next command depends on enabled SSH.

Reply:
yep,  good point, the next all commands depends on enables SSH service. 
So, during  call test_network_interface(),  ensure guest ssh service firstly was so important.

Reply:
I don't understand.

Reply:
Refer to `test_network_interface()`,  all commands depend on SSH service as workable status under guest systems, means that keep connection both on vm host and guest systems together depend on enabled SSH service here. 
So, we'd better to ensure guest system SSH service as UP (or enabled) status to call `script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180);` here. 
If so, we can continue the follow commands under `test_network_interface()`
if not,  we need to debug this SSH problem under guest system.

Reply:
OU! You're removing the condition, not adding it!
I'm very sorry, it's correct then. LGTM.